### PR TITLE
fix: remove null values before processing

### LINKED ISSUE #165: TypeError: Cannot read properties of null (reading 'toLowerCase')
Sentry Issue: [STAMP-1J](https://snapshot-labs.sentry.io/issues/4868263621/?referrer=github_integration)

```
TypeError: Cannot read properties of null (reading 'toLowerCase')
  File "/workspace/dist/src/addressResolvers/utils.js", line 56, in <anonymous>
    const inputLc = input.map(i => i.toLowerCase());
  File "/workspace/dist/src/addressResolvers/utils.js", line 56, in mapOriginalInput
    const inputLc = input.map(i => i.toLowerCase());
  File "/workspace/dist/src/addressResolvers/index.js", line 69, in lookupAddresses
    return (0, utils_1.mapOriginalInput)(addresses, result);
  File "/workspace/dist/src/api.js", line 25, in <anonymous>
    result = await (0, addressResolvers_1.lookupAddresses)(params);
...
(2 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by wa0x6e**
Fix #165 
Fix [STAMP-1J](https://snapshot-labs.sentry.io/issues/4868263621/?referrer=github_integration)

Remove null values before manipulation

### CODE REVIEW

File: src/addressResolvers/utils.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export function mapOriginalInput(
   input: string[],
   results: Record<string, string>
 ): Record<string, string> {
-  const inputLc = input.map(i => i.toLowerCase());
+  const inputLc = input.filter(a => a).map(i => i.toLowerCase());
```
Comment (ChaituVR) [inline src/addressResolvers/utils.ts:60]: Index of `input` is being used with inputLc's index in return 

```
return withoutEmptyValues(
    Object.fromEntries(
      inputLc.map((key, index) => {
        return [input[index], resultLc[key]];
      })
    )
  );
  ```
  maybe there can be differences?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/addressResolvers/utils.ts (hunk 1 above)
Comment (wa0x6e) [reply src/addressResolvers/utils.ts:60]: true

Comment (wa0x6e) [reply src/addressResolvers/utils.ts]: Should be better now

**Review by ChaituVR — APPROVED** [review]
utAck

**Pull request merged**